content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -5,48 +5,146 @@ description: >
   Learn how to install the Armory Agent in your Kubernetes and Armory Enterprise environments.
 weight: 2
 ---
-![Proprietary](/images/proprietary.svg)
+![Proprietary](/images/proprietary.svg) This feature requires an Armory licensed entitlement.
 
-## Compatibility matrix
+## Overview
 
-{{< include "agent/agent-compat-matrix.md" >}}
-
-The Agent consists of a service deployed as a Kubernetes `Deployment` and a plugin to Spinnaker's Clouddriver service. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
+The Agent consists of a plugin to Spinnaker's Clouddriver service and a K8s deployment that connects to Clouddriver. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
 
 ## {{% heading "prereq" %}}
 
-* This guide is for experienced Kubernetes and Armory Enterprise users.
+* This guide assumes you're using the Armory Operator to install Spinnaker, with the Kustomize method from the [spinnaker-kustomize-patch repo](https://github.com/armory/spinnaker-kustomize-patches).
+
 * You have read the Armory Agent [overview]({{< ref "armory-agent" >}}).
-* You have a Redis instance. The Agent uses Redis to coordinate between Clouddriver replicas.
 * You have configured Clouddriver to use MySQL or PostgreSQL. See the {{< linkWithTitle "clouddriver-sql-configure.md" >}} guide for instructions.
+* You have Spinnaker installed in one cluster, and an additional cluster to serve as your target deployment cluster.
+
+### Compatibility matrix
+
+{{< include "agent/agent-compat-matrix.md" >}}
+
+### Networking requirements
 
-## Networking requirements
+Communication from the Agent to Clouddriver occurs over gRPC port 9091.
 
-Communication between Clouddriver and the Agent must be `http/2`. `http/1.1` is *not* compatible and causes communication issues between Clouddriver and the Agent.  
+### Kubernetes permissions required by the Agent
 
-## Kubernetes permissions needed by the Agent
+The Agent can use a kubeconfig file loaded as a K8s secret (This is only appropriate when the agent runs in a separate cluster from the target cluster), or as a service account in the cluster it resides in. Running Agent in the target cluster with a service account is the preferred model. 
```

Comment:
k8s -> Kubernetes.

> Running Agent in the target cluster with a service account is the preferred model.
This is not true. There are plenty of reasons to prefer infrastructure or service mode: easier to manage credentials if they are generated, policies forbidding agents, difficulties upgrading Agents as we don't have a good answer for that yet.

Reply:
ok, it's a matter of opinion I think. I can just remove the preferred statement.

Reply:
Maybe instead of preferred is to say `the common model we see` and then elaborate that there are additional advanced configurations to enable scalability.

Reply:
sounds good. actually, i think more now that this is a main doc topic, then we just ommit the preferred here. so it stays as a very clear quick start